### DISCUSSION TITLE
Cant edit certain fields to german language

### DISCUSSION THREAD

**Discussion by morrissey12345** [woocommerce/woocommerce#50358] [Checkout Flow]
Looks like the field "Use same adress for billing, Order summary, add suite are not editable. Even in Loco translate i cant find the sentences. Are they hardcoded ?

**Comment by senadir** [comment]
No, they're translated via JSON, which LOCO might not fully support. However, the values are translated in German for me

<img width="443" alt="image" src="https://github.com/user-attachments/assets/57f3ce42-ad6f-44f0-bc93-4a6bd8974a32">

Using a plugin like "Say What" works fine for me

<img width="1059" alt="image" src="https://github.com/user-attachments/assets/fc8d95c4-b7e0-4612-a8e4-72d63ac3b670">


<img width="248" alt="image" src="https://github.com/user-attachments/assets/41e08ae5-57a0-468e-9b6d-d0485e56a6ba">
